app/javascript/styles/mastodon/forms.scss

```diff
@@ -110,6 +110,12 @@ code {
         color: $primary-text-color;
         display: block;
         width: auto;
+
+        &.boolean {
```

Comment:
I do not think that nesting the `.boolean` class inside itself is necessary.

Reply:
This is hard to see from the nested scss here -- but I think the one higher up is targetting the parent input element and this style is targetting the `.label_input label.boolean` element that has these little badges in it.

Reply:
Thanks for clarifying and sorry for the noise :man_facepalming: 

FWIW the nesting was only a small part of the problem. What tripped me up is the generous amount of classes that `simple_form` adds all over the place. I was an avid user of `simple_form` a long time ago, but now it seems I am not used to that anymore.